planner/core/common_plans.go

```diff
@@ -368,6 +368,7 @@ type Deallocate struct {
 }
 
 // Show represents a show plan.
+// TODO: make as a logical data source or table dual.
```

Comment:
Why not just do it now?

Reply:
i want to fix the unit-tests in my hands ASAP previously. :joy: